src/main/java/com/github/pedrovgs/androidwifiadb/adb/ADB.java

```diff
@@ -28,6 +29,7 @@
   private final CommandLine commandLine;
   private final ADBParser adbParser;
   private Project project;
+  private boolean isTCPEnabled = false;
```

Comment:
I've fixed this using adb command. 
We don't need anymore boolean property to check if the tcpip port was enabled.